cmd/stakercli/pop/pop.go

```diff
@@ -67,7 +75,7 @@ var generateCreatePopCmd = cli.Command{
 		cli.StringFlag{
 			Name:  btcNetworkFlag,
 			Usage: "Bitcoin network on which staking should take place (testnet3, mainnet, regtest, simnet, signet)",
-			Value: "testnet3",
+			Value: "main",
```

Comment:
not a blocker, but imo commands by defaults should use testnets and act of acting on mainnet should be concious choice of the operator